x-pack/solutions/observability/plugins/apm/public/components/shared/trace_waterfall/trace_waterfall.stories.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+import { TraceWaterfall } from '.';
+import { traceOtelSample } from './mock/trace_otel_sample';
+import { traceSample } from './mock/trace_sample';
+import { MockApmPluginStorybook } from '../../../context/apm_plugin/mock_apm_plugin_storybook';
+import type { TraceItem } from '../../../../common/waterfall/unified_trace_item';
+
+const stories: Meta = {
+  title: 'UnifiedTraceWaterfall',
+  component: TraceWaterfall,
+  decorators: [
+    (StoryComponent) => (
+      <MockApmPluginStorybook>
+        <StoryComponent />
+      </MockApmPluginStorybook>
+    ),
+  ],
+};
+
+export default stories;
+
+export const ExampleClockSkew: StoryFn<{}> = () => {
+  return (
+    <TraceWaterfall
+      onClick={(params) => {
+        // eslint-disable-next-line no-console
+        console.log('##### Clicked:', params);
+      }}
+      onErrorClick={(params) => {
+        // eslint-disable-next-line no-console
+        console.log('##### Error clicked:', params);
+      }}
```

Comment:
Storybook provides a helper function do to that instead of `console.log`

You'll need to:

`import { action } from '@storybook/addon-actions';`

```suggestion
      onClick={action('onClick')}
      onErrorClick={action('onErrorClick')}
```